README.md

```diff
@@ -16,6 +16,18 @@ It's the spiritual successor to https://github.com/helix-collective/camus
   - A stated design goal is each exposed command isn't meant to be a complex abstraction, but rather a convenience. If required, a support engineer should be able to execute the commands by hand (albiet it might be a painful and long process)
   - A second stated design goal is rollbacks and roll-forwards should be fast, easy and have 0 downtime. By default the various connect/disconnect commands cause the nginx frontend proxy to reconfigure and reload, forwarding traffic to the specified running deploy.
 
+# Requirements:
+ - Configuration file: see docs/example_deploy.json for an example and adl/config.adl for specification in ADL. defaults to ../etc/hx-deploy-tool.json
+ - AWS Credentials: see https://aws.amazon.com/blogs/security/a-new-and-standardized-way-to-manage-credentials-in-the-aws-sdks/ 
+ - S3 buckets with necessary permissions for storing state and retrieving releases
+ - Folders with necessary permissions for writing logs
+ -  Release Binaries:
+    Stored in S3 in a .zip file
+    Contains - binary to be deployed
```

Comment:
Correction - the zip packages stored in s3 are not necessarily binaries - they can be (and generally are) just scripts.

Reply:
@timbod7 you approved, want me to update the docco in the next commit or this one?